.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: release
+
+on:
+  push:
+    tags:
```

Comment:
Perhaps we should also specify master branch constraint? Just in case we accidentally tag a branch commit.

Reply:
I tried that, unfortunately it turns out that if you set multiple triggers, they are matched with a logical 'OR'. This is not explicitly documented, but I verified it on my test repo.

Reply:
Well that is no fun :smirk: